index.html

```diff
@@ -121,51 +121,109 @@ <h2>Becoming a full member of a section</h2>
   This node is going to try to be accepted as candidate until it receives a Rpc::NodeConnected to complete this stage successfully.<br/>
   This node is going to perform the resource proof until it receives a Rpc::NodeApproval to complete this stage successfully.<br/>
   If the node is not accepted, after time out, it will try another section as a new node.
+  <button class="collapsible">TimeoutResendInfo</button>
+  <div class="content">
+      <p>Timeout triggered to resend messages that have been lost: Either Rpc::CandidateInfo or Rpc::ResourceProofResponse
+      </p>
+  </div>
+  <button class="collapsible">TimeoutConnectRefused</button>
+  <div class="content">
+      <p>Timeout to stop trying to connect if so much time as elapsed that we will not be able to succeed.
+      </p>
+  </div>
+  <button class="collapsible">TimeoutProofRefused</button>
+  <div class="content">
+      <p>Timeout to stop trying to resource proof if so much time as elapsed that we will not be able to succeed.
+      </p>
+  </div>
+  <button class="collapsible">get_next_resource_proof_part</button>
+  <div class="content">
+      <p>Return the next part of the resouce proof for the given elder.<br/>
+      inputs:<br/>
+          - elder name<br/>
+      </p>
+  </div>
+  <button class="collapsible">get_resend_resource_proof_part</button>
+  <div class="content">
+      <p>Return the last part of the resouce proof for the given elder that we sent.<br/>
+      inputs:<br/>
+          - elder name<br/>
+      </p>
+  </div>
+  <button class="collapsible">get_resource_proof_elders</button>
+  <div class="content">
+      <p>Return all the elder we are currently sending resource proofs too.
+      </p>
+  </div>
+  <button class="collapsible">NEED_RESEND_PROOFS</button>
+  <div class="content">
+      <p>Collection of elder name we have not sent a ResourceProofResponse during the timeout.
```

Comment:
I think "name" is superfluous here, and we didn't use it in the previous collapsible sections.  If you prefer to keep it, it should be changed to "names" :)